be1-go/popcha/server.go

```diff
@@ -37,7 +37,7 @@ const (
 	tokenLifeTimeHour = 24
 
 	// qrSize size of QRCode SVG
-	qrSize = 10
+	qrSize = 9
```

Comment:
this was change for aesthetic purpose (zooming too much with size 10 would make the QR Code way out of frame)